packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkImplementation.kt

```diff
@@ -546,6 +561,8 @@ class DdSdkImplementation(
         internal const val DD_NATIVE_VIEW_TRACKING = "_dd.native_view_tracking"
         internal const val DD_NATIVE_INTERACTION_TRACKING = "_dd.native_interaction_tracking"
         internal const val DD_SDK_VERBOSITY = "_dd.sdk_verbosity"
+        internal const val DD_RUM_ENABLE_LOGS = "_dd.enable_rum_for_logs"
+        internal const val DD_TRACES_ENABLE_LOGS = "_dd.enable_traces_for_logs"
```

Comment:
let's align names with values (and this is the naming used in `DdSdkReactNative.tsx` already anyway):

```suggestion
        internal const val DD_ENABLE_RUM_FOR_LOGS = "_dd.enable_rum_for_logs"
        internal const val DD_ENABLE_TRACES_FOR_LOGS = "_dd.enable_traces_for_logs"
```